modules/articles/client/controllers/list-articles.client.controller.js

```diff
@@ -9,7 +9,45 @@
 
   function ArticlesListController(ArticlesService) {
     var vm = this;
+    var rightNow = new Date(Date.now());
 
-    vm.articles = ArticlesService.query();
+    vm.pageChanged = pageChanged;
+
+    init();
+
+    function init() {
+      vm.itemsPerPage = 20;
+      vm.currentPage = 1;
+
+      vm.statistics = ArticlesService.statistics();
+
+      find();
+    }
+
+    function pageChanged() {
```

Comment:
This seems like it's just aliasing, and then you alias this into vm, so why not just do it straight to vm and save on code and memory.